### PR TITLE
Add recipe for dials-dependencies

### PR DESCRIPTION
**Pull request by Anthchirp**
This is a metapackage to assist users in installing [DIALS](https://dials.github.io/), which itself currently can not be bundled via conda/pip.

### CODE REVIEW

File: recipes/dials-dependencies/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is convenient
```
Comment (marcelotrevisani) [inline recipes/dials-dependencies/meta.yaml]: Please remove the unnecessary comments

**Comment by marcelotrevisani** [comment]
Why DIAS cannot be installed using ``conda`` ?

**Comment by Anthchirp** [comment]
The main reason holding us back is that DIALS uses another large scientific software library that has a number of complex dependencies, contains its own build environment management and has a very involved build process. I'd be hesitant to make any promises but with the way things are going currently it may be realistic to move to a conda-installable package, for DIALS possibly even wheel packages, within 2 years or so.

(As is common with many old scientific projects we are also stuck with Python 2.7, so a good time to make big changes)

**Comment by marcelotrevisani** [comment]
> The main reason holding us back is that DIALS uses another large scientific software library that has a number of complex dependencies, contains its own build environment management and has a very involved build process. I'd be hesitant to make any promises but with the way things are going currently it may be realistic to move to a conda-installable package, for DIALS possibly even wheel packages, within 2 years or so.
> 
> (As is common with many old scientific projects we are also stuck with Python 2.7, so a good time to make big changes)

got it! But if someday you want to give a try you can ping us to help you.  :)

**Pull request merged**